File: 2012/submissions/Pat Smuk/1/Go/Problem1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func readln(r *bufio.Reader) string {
    var (
        isPrefix bool = true
        err error     = nil
        line, ln []byte
    )

    for isPrefix && err == nil {
        line, isPrefix, err = r.ReadLine()
        ln = append(ln, line...)
    }

    return string(ln)
}

func main() {
	fmt.Print("Enter the word to search for: ")	
	var search string
	fmt.Scanln(&search)
	
	fmt.Print("Enter the text to search: ")
	in := bufio.NewReader(os.Stdin)
	var text string
	text = readln(in)
	
	fmt.Printf("First index of \"%s\" in text: %d\n", search, strings.Index(text, search))
}

File: 2012/submissions/Pat Smuk/1/C#/Problem1.cs
using System;

namespace CSClubContest2012CS
{
    class Problem1
    {
        static void Main(string[] args)
        {
            Console.Write("Enter the word to search for: ");
            string word = Console.ReadLine();

            Console.Write("Enter the text to search: ");
            string text = Console.ReadLine();

            Console.WriteLine("First index of \"" + word + "\" in text: " + text.IndexOf(word));
            Console.ReadKey();
        }
    }
}
